prod-stack.py

```diff
@@ -723,6 +723,13 @@
                 ],
                 'depends': ['webhooks', 'legacyhooks']
             },
+            {
+                'name': 'TicketCloser',
+                'command': 'ticket-closer',
+                'env': [],
+                'secrets': ['GH_Token'],
+                'schedule': 'rate(1 day)',
+            },
         ]
```

Comment:
Today I Learned :tm:

Reply:
Hahaha, yeah don't worry that took some time to figure out. The documentation wasn't as clear as it could have been regarding exactly what IAM permissions were needed. There is more that can be done to make this all a bit more automatic, but for now 'Trap For Young Players&trade;'

Reply:
I think of it as, you've done a good job insulating me from those complexities :1st_place_medal: